.github/workflows/runner-e2e-tests-codeceptjs.yml

```diff
@@ -111,52 +107,62 @@ jobs:
           repository: percona/pmm-ui-tests
           path: ./pmm-ui-tests
 
+      - name: Checkout qa-integration repo
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ env.PMM_QA_BRANCH }}
+          repository: Percona-Lab/qa-integration
+          path: ./qa-integration
+
       - name: Setup tools
         run: |
-          npm install -g bats
           sudo apt-get install -y apt-transport-https ca-certificates dirmngr ansible libaio1 libaio-dev libnuma-dev libncurses5 socat sysbench
           sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 8919F6BD2B48D754
           echo "deb https://packages.clickhouse.com/deb stable main" | sudo tee \
               /etc/apt/sources.list.d/clickhouse.list
           sudo apt-get update
           sudo apt-get install -y clickhouse-client
+          sudo apt-get install -y ansible
           sudo curl -s https://raw.githubusercontent.com/datacharmer/dbdeployer/master/scripts/dbdeployer-install.sh | bash
           ls -la
-          git clone -b ${{ env.PMM_QA_BRANCH }} https://github.com/percona/pmm-qa
-          ls -la
-          pushd pmm-qa
+          pushd qa-integration
           wget https://raw.githubusercontent.com/Percona-QA/percona-qa/master/get_download_link.sh
           chmod +x get_download_link.sh
```

Comment:
Please feel free to remove use of get_download_link.sh, we are not dependent on it anymore.